android/docs/help/about/adware.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Does Keyman include adware or spyware?
+---
+
+No!
+
+We never have and never will include adware or spyware components. 
+Keyman does contact the keyman.com website periodically to check for updates for both Keyman and the keyboards you have installed. 
+In this process, the only information communicated is the current version of Keyman and the version of the keyboards.
+
+When [enabling Keyman as a system keyboard](../start/enabling-system-keyboard), Android displays the following warning, which is standard for any system keyboard.
+
+![](../android_images/attention.png)
```

Comment:
```suggestion
![](../android_images/attention.png)

> Attention: This input method may be able to collect all the text you 
> type, including personal data like passwords and credit card numbers.
> It comes from the app Keyman. Use this input method? Cancel / OK

The Keyman app does not record the keystrokes you type and does not send keystrokes off-device, unlike many other keyboard apps. All predictive text suggestions are entirely on your device.

Keyman is an open source application, and you are most welcome to inspect the source code at https://github.com/keymanapp/keyman.
```

Should also link to privacy.md and perhaps in that reference this doc too. (privacy already mentions sentry, I see)